src/app/containers/PodcastPromo/index.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useContext } from 'react';
+import path from 'ramda/src/path';
+import { string } from 'prop-types';
+import PodcastPromo from '@bbc/psammead-podcast-promo';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import ImageWithPlaceholder from '#containers/ImageWithPlaceholder';
+
+const Promo = ({
+  title,
+  brandTitle,
+  brandDescription,
+  imageSrc,
+  imageAlt,
+  linkHref,
+  linkText,
+}) => {
+  const { podcastPromo, script, service } = useContext(ServiceContext);
+  const podcastPromoTitle = title || path(['title'], podcastPromo);
+  const podcastBrandTitle = brandTitle || path(['brandTitle'], podcastPromo);
+  const description =
+    brandDescription || path(['brandDescription'], podcastPromo);
+  const img = imageSrc || path(['image', 'src'], podcastPromo);
+  const alt = imageAlt || path(['image', 'alt'], podcastPromo);
+  const url = linkHref || path(['linkLabel', 'href'], podcastPromo);
+  const label = linkText || path(['linkLabel', 'text'], podcastPromo);
+
+  const showPromo = [
+    podcastBrandTitle,
+    podcastPromoTitle,
+    description,
+    img,
+    alt,
+    url,
+    label,
+  ].every(Boolean);
+  if (!showPromo) {
+    return null;
+  }
+
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```
Hella minor thing, may not be preferred either. Could we just use a fragment here?

Reply:
I don't think we need a div or a fragment

Reply:
Great point. 🗑️